client/basePaths.ts

```diff
@@ -18,29 +19,37 @@ export const BasePaths = {
   Assessment: {
     root: (
       countryIso: CountryIso | defaults.countryIso | string = defaults.countryIso,
-      assessmentType: AssessmentName | defaults.assessmentType = defaults.assessmentType
-    ) => _generate('assessment', countryIso, assessmentType),
+      assessmentName: AssessmentName | defaults.assessmentName = defaults.assessmentName
+    ) => _generate('assessment', countryIso, assessmentName),
 
     section: (
       countryIso: CountryIso | defaults.countryIso = defaults.countryIso,
-      assessmentType: AssessmentName | defaults.assessmentType = defaults.assessmentType,
+      assessmentName: AssessmentName | defaults.assessmentName = defaults.assessmentName,
       section: string | defaults.section = defaults.section
-    ) => _generate('assessment', countryIso, assessmentType, section),
+    ) => _generate('assessment', countryIso, assessmentName, section),
     dataDownload: (
       countryIso: CountryIso | defaults.countryIso = defaults.countryIso,
-      assessmentType: AssessmentName | defaults.assessmentType = defaults.assessmentType
-    ) => _generate('assessment', countryIso, assessmentType, 'dataDownload'),
+      assessmentName: AssessmentName | defaults.assessmentName = defaults.assessmentName
+    ) => _generate('assessment', countryIso, assessmentName, 'dataDownload'),
     print: (
       countryIso: CountryIso | defaults.countryIso = defaults.countryIso,
-      assessmentType: AssessmentName | defaults.assessmentType = defaults.assessmentType,
+      assessmentName: AssessmentName | defaults.assessmentName = defaults.assessmentName,
       tables = false
-    ) => _generate(countryIso, assessmentType, 'print', tables && 'tables'),
+    ) => _generate(countryIso, assessmentName, 'print', tables && 'tables'),
   },
   Login: {
     root: () => '/login',
     resetPassword: () => `/login/resetPassword`,
     invitation: () => '/login/invitation',
   },
+  OriginalDataPoint: {
+    root: (
+      countryIso: CountryIso | defaults.countryIso = defaults.countryIso,
+      assessmentName: AssessmentName | defaults.assessmentName = defaults.assessmentName,
+      cycleName = defaults.cycleName,
+      odpId = ':odpId'
+    ) => _generate(countryIso, assessmentName, cycleName, 'originalDataPoint', odpId),
```

Comment:
should we prefix with assessment like other assessment paths?
if not, should we prefix instead with originalDataPoint?

currently looks like:
http://localhost:9000/FIN/fra/2025/originalDataPoint/72

Reply:
it should follow assessment convention. then yes.
and they should be: /:countryIso/assessments/:name/cycles/:cycleName/originalDataPoint/:odpId

Reply:
call the key, one, not root

Reply:
I think we need to revisit the paths in general
For client side, I think 'root' makes more sense?
We also have other paths under OriginalDataPoint (section, tab)

Renamed for now